crates/core/src/parser/parser.rs

```diff
@@ -78,6 +78,16 @@ pub fn parse(text: &str, options: &Language) -> Result<Vec<Box<dyn Renderable>>>
     Ok(renderables)
 }
 
+/// Given a `Variable` as a string, parses it into a `Variable`.
+pub fn parse_variable_from_text(text: &str) -> Result<Variable> {
```

Comment:
Let's remove the `_from_text`

Reply:
(that requires renaming `parse_variable` which I'm fine with, no strong preference on that name)

Reply:
Sounds good! I'll take care of this later today, and I'll also learn some Git magic to try and merge our PRs together. Thanks for your help!